.github/scripts/hermetic_library_generation.sh

```diff
@@ -101,8 +94,15 @@ docker run \
   gcr.io/cloud-devrel-public-resources/java-library-generation:"${image_tag}" \
   --baseline-generation-config-path="${workspace_name}/${baseline_generation_config}" \
   --current-generation-config-path="${workspace_name}/${generation_config}"
+
 # commit the change to the pull request.
-git add java-* pom.xml gapic-libraries-bom/pom.xml versions.txt
+if [[ $(basename "$(pwd)") == "google-cloud-java" ]]; then
```

Comment:
Do we have to differentiate between google-cloud-java and java-* repos? Can we just do `git add .`?